app/navigation/Navigator.scala

```diff
@@ -77,11 +81,15 @@ class Navigator @Inject()() {
     case _ => routes.ImporterHasEoriController.onPageLoad(NormalMode)
   }
 
-  private def getImporterName(answers: UserAnswers): Call = answers.get(ClaimantTypePage) match {
-    case Some(ClaimantType.Representative)  => routes.AgentImporterAddressController.onPageLoad(NormalMode)
-    case _ => routes.ImporterAddressController.onPageLoad(NormalMode)
+  private def getDeclarantName(answers: UserAnswers): Call = answers.get(ClaimantTypePage) match {
+    case Some(ClaimantType.Importer)  =>  routes.DoYouOwnTheGoodsController.onPageLoad(NormalMode)
+    case _ => ???
```

Comment:
same here like above comment.

Reply:
Because it is not yet implemented . This is scala short hand for that